### ISSUE TITLE
Repo *** not found in your editor in pull requests - VSCode

### ISSUE THREAD

**Issue by lc-4918** [TeamCodeStream/codestream#1300]
- IDE Version: VSCode 1.74.2
- OS Version: Linux Ubuntu 22.04.1 LTS

Steps to Reproduce:

1. Open local repository (File > Open folder)
2. Connect to codestream with gitlab authentication
3. Open a PR

It tells me to open the current opened repo !
![image](https://user-images.githubusercontent.com/46907200/210550248-76b303ae-f938-431d-af1e-0ecaedac1035.png)

Log:
[codestream.log](https://github.com/TeamCodeStream/codestream/files/10343795/codestream.log)

Bug related with #1274 , #1171 and #771

**Comment by bcanzanella** [comment]
@lc-4918 could you post the output of `git remote -v` here? thanks.

**Comment by bcanzanella** [comment]
@lc-4918 do you require a VPN or a password to fetch from origin?

can you try running `git fetch --all` from your repo root? thanks.

**Comment by lc-4918** [comment]
@bcanzanella I require a password to fetch from origin (no VPN).

I tried:

1. run `git fetch -all`
2. refresh PR

Then, it works! I could see the PR's changed files and the bug's warning message has disappeared.

Today, I faced again the bug materialized by the warning message.
I ran again `git fetch -all` and refresh PR display. And The bug disappeared.

Maybe the bug occurs when one local branch isn't updated...

With this workaround, now I can review in VSCode. Thanks @bcanzanella
